benchmark-service/benchmark/benchmark.py

```diff
@@ -53,11 +53,11 @@
     oauth_token = f.read().strip()
 
 
-def get_benchmarks(app, file_path):
+async def get_benchmarks(app, file_path):
     log.info(f'get_benchmarks file_path={file_path}')
-    gs_reader = app['gs_reader']
+    gs_reader: aiogoogle.GoogleStorageAsyncFS = app['gs_reader']
```

Comment:
Why does this need to be a Google FS? If it's a temporary requirement it might be better to treat it as non-google-specific wherever possible and assert that it is a `GoogleStorageAsyncFS` in the few places where it needs to be until those are abstracted.